src/phpDocumentor/Configuration/ValueObjects/VersionObject.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @link https://phpdoc.org
+ */
+
+namespace phpDocumentor\Configuration\ValueObjects;
+
+use phpDocumentor\Dsn;
+use phpDocumentor\Path;
+
+class VersionObject
```

Comment:
I think this class should be named `phpDocumentor\Configuration\VersionSpecification`  There is no need to have a separate `ValueObjects` namespace for this.

Reply:
Done :)